tests/fs/test_mount.c

```diff
@@ -27,15 +27,15 @@ int main() {
     } catch (e) {
       ex = e;
     }
-    assert(ex instanceof FS.ErrnoError && ex.errno === 2); // ENOENT
+    assert(ex instanceof FS.ErrnoError && ex.errno === 44); // ENOENT
```

Comment:
Why not use the macros here?

Reply:
Hmm, I guess the macros would work as this EM_ASM code, so the C preprocessor does run on it. May make sense as a followup, yeah.